src/index.js

```diff
@@ -2,16 +2,11 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
```

Comment:
- [ ] Got an error while running the project, that App wasn't imported properly, kindly update the path.
- [ ] No books are displayed on the page, kindly take a look at it.